docs/known_issues.md

```diff
@@ -4,11 +4,6 @@ title: Known issues
 ---
 
 # Known issues
-
-## form_for compatibility
-
-ViewComponent is [not currently compatible](https://github.com/github/view_component/issues/241) with `form_for` helpers.
```

Comment:
What about keeping it with a link to the Forms page?

Reply:
Let me know what you think of this :)

Reply:
Great!